Language/Functions/Random Numbers/randomSeed.adoc

```diff
@@ -30,7 +30,7 @@ Conversely, it can occasionally be useful to use pseudo-random sequences that re
 
 [float]
 === Parameters
-`long`, `int` - pass a number to generate the seed.
+`seed` - number to initialize the pseudo-random sequence (long).
```

Comment:
It's `unsigned long`:
https://github.com/arduino/ArduinoCore-avr/blob/58006613a7c858417881744419c5da54cc21a401/cores/arduino/Arduino.h#L252
```c++
void randomSeed(unsigned long);
```
*In SAM and SAMD boards cores, `uint32_t`